### ISSUE TITLE
Android 17.0 is remuxing for no reasons.

### ISSUE THREAD

**Issue by Alliyance** [jellyfin/jellyfin-androidtv#3838]
### Describe the bug

I did open a ticket a month ago for that same reason with Chromcast and my ticket was closed, said it was the client side issue. With the new update0.17.0, It does the same thing with Firetv with or without a soundbar. I don't think the audio is the issue and either the video. It seem that the container MKV isn't compatable anymore. 

### Logs

[2024-08-05 17:23:46.736 +00:00] [INF] [71] MediaBrowser.MediaEncoding.Transcoding.TranscodeManager: "/usr/bin/ffmpeg" "-analyzeduration 200M -probesize 1G -fflags +genpts -f matroska,webm  -i file:\"/home/alliyance/files/Movies/5lbs.of.Pressure.2024.mkv\" -map_metadata -1 -map_chapters -1 -threads 0 -map 0:0 -map 0:1 -map -0:s -codec:v:0 copy -bsf:v h264_mp4toannexb -start_at_zero -codec:a:0 copy -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 6 -hls_segment_type mpegts -start_number 0 -hls_segment_filename \"/dev/shm/1011/jellyfin/transcodes/3c3969ff1470b162179404cea816937d%d.ts\" -hls_playlist_type vod -hls_list_size 0 -y \"/dev/shm/1011/jellyfin/transcodes/3c3969ff1470b162179404cea816937d.m3u8\""
[2024-08-05 17:23:47.703 +00:00] [INF] [71] MediaBrowser.MediaEncoding.Subtitles.SubtitleEncoder: "/usr/bin/ffmpeg" "-i \"/home/alliyance/files/Movies/5lbs.of.Pressure.2024.mkv\" -copyts -map 0:5 -an -vn -c:s copy \"/home/alliyance/.config/jellyfin/data/data/subtitles/6/68d253ea-e308-5680-bce4-625092cbb1e7.srt\""
[2024-08-05 17:24:08.196 +00:00] [INF] [75] MediaBrowser.MediaEncoding.Subtitles.SubtitleEncoder: ffmpeg subtitle extraction completed for "/home/alliyance/files/Movies/5lbs.of.Pressure.2024.mkv" to "/home/alliyance/.config/jellyfin/data/data/subtitles/6/68d253ea-e308-5680-bce4-625092cbb1e7.srt"
[2024-08-05 17:24:09.306 +00:00] [INF] [131] MediaBrowser.MediaEncoding.Transcoding.TranscodeManager: FFmpeg exited with code 0
[2024-08-05 17:25:30.619 +00:00] [INF] [39] MediaBrowser.MediaEncoding.Transcoding.TranscodeManager: Deleting partial stream file(s) "/dev/shm/1011/jellyfin/transcodes/3c3969ff1470b162179404cea816937d.m3u8"
[2024-08-05 17:25:32.994 +00:00] [INF] [163] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Android TV" "0.17.0" playing "5lbs of Pressure". Stopped at "60700" ms
[2024-08-05 17:27:36.669 +00:00] [INF] [112] Jellyfin.Server.Implementations.Users.UserManager: Authentication request for "alliyance" has succeeded.
[2024-08-05 17:27:36.669 +00:00] [INF] [112] Emby.Server.Implementations.Session.SessionManager: Current/Max sessions for user "alliyance": 0/0
[2024-08-05 17:27:36.670 +00:00] [INF] [112] Emby.Server.Implementations.Session.SessionManager: Creating new access token for user 92e149af-039b-40e4-a587-ddbc73aac010
[2024-08-05 17:27:36.840 +00:00] [INF] [78] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "127.0.0.1" request

### FFmpeg logs

```shell
/usr/bin/ffmpeg -analyzeduration 200M -probesize 1G -fflags +genpts -f matroska,webm  -i file:"/home/alliyance/files/Movies/5lbs.of.Pressure.2024.mkv" -map_metadata -1 -map_chapters -1 -threads 0 -map 0:0 -map 0:1 -map -0:s -codec:v:0 copy -bsf:v h264_mp4toannexb -start_at_zero -codec:a:0 copy -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 6 -hls_segment_type mpegts -start_number 0 -hls_segment_filename "/dev/shm/1011/jellyfin/transcodes/3c3969ff1470b162179404cea816937d%d.ts" -hls_playlist_type vod -hls_list_size 0 -y "/dev/shm/1011/jellyfin/transcodes/3c3969ff1470b162179404cea816937d.m3u8"
```

### Media info of the file

```shell
5lbs.of.Pressure.2024
Container: mkv
Path: /home/alliyance/files/Movies/5lbs.of.Pressure.2024.mkv
Size: 4440 MB

Subtitle
Title: Forced - Fre - SUBRIP
Language: fre
Codec: SUBRIP
Default: No
Forced: Yes
External: No

Video
Title: 1080p H264 SDR
Codec: H264
AVC: Yes
Profile: High
Level: 41
Resolution: 1920x800
Aspect ratio: 2.40:1
Interlaced: No
Framerate: 23.976025
Bitrate: 5634 kbps
Bit depth: 8 bit
Video range: SDR
Video range type: SDR
Pixel format: yuv420p
Ref frames: 1
NAL: 4

Audio
Title: English - AAC - 5.1 - Default
Language: eng
Codec: AAC
Profile: LC
Layout: 5.1
Channels: 6 ch
Bitrate: 445 kbps
Sample rate: 48000 Hz
Default: Yes
Forced: No
External: No

Audio
Title: Fre - AAC - 5.1
Language: fre
Codec: AAC
Profile: LC
Layout: 5.1
Channels: 6 ch
Bitrate: 445 kbps
Sample rate: 48000 Hz
Default: No
Forced: No
External: No

Subtitle
Title: Full - English - SUBRIP
Language: eng
Codec: SUBRIP
Default: No
Forced: No
External: No

Subtitle
Title: Full - Fre - SUBRIP
Language: fre
Codec: SUBRIP
Default: No
Forced: No
External: No
```

### Application version

0.17.0

### Device information

Firetv stick 4K

### Android version

firetv

### Jellyfin server version

10.9.7

**Comment by myanesp** [comment]
Hi!

Can you try to go to Settings (on Android TV app), and change Audio Output to "Direct" instead of "Remuxing to stereo" and see if that helps to avoid remuxing?

That did the trick for me

**Comment by Alliyance** [comment]
Yeah that fixed it. Is it possible with the next update to make it Direct by default?

**Comment by nielsvanvelzen** [comment]
The default was deliberately changed in 0.17 to prefer downmixing instead. It did prefer direct play before. The reason was that we saw a bunch of issues caused by those defaults. Considering the amount of reports now that downmixing is the default I'm not sure which option is the best. I'll see what I can do about that.

Closing this issue for now though, as this is currently the intended behavior.
